src/keyword-value.js

```diff
@@ -22,7 +22,7 @@
     this.cssString = value;
   }
 
-  KeywordValue.StyleValueKeyword = ['initial', 'inherit', 'revert', 'unset'];
+  KeywordValue.StyleValueKeyword = ['auto', 'initial', 'inherit', 'revert', 'unset'];
```

Comment:
Perhaps this should be GlobalKeyword, and other things that refer to it for parsing/validation purposes should also refer to "isGlobalKeyword" and such. I think the keywords for particular types should go in the classes used for those purposes. It's probably still worth discussion of what the structure should look like for something like "speech-rate" which takes 

<number> | x-slow | slow | medium | fast | x-fast | faster | slower | inherit

since it doesn't look like there should be a SpeechRateValue class...